skins/oasis/modules/OasisController.class.php

```diff
@@ -91,13 +91,24 @@ public static function JsAtBottom(){
 	}
 
 	public function executeIndex($params) {
-		global $wgOut, $wgUser, $wgTitle, $wgRequest, $wgEnableAdminDashboardExt, $wgOasisThemeSettings;
+		global $wgOut, $wgUser, $wgTitle, $wgRequest, $wgEnableAdminDashboardExt, $wgOasisThemeSettings,
+		$wgWikiaMobileSmartBannerConfig;
 
 		wfProfileIn(__METHOD__);
 
 		//Add Smart banner for My Wikia App
 		//See: https://wikia-inc.atlassian.net/browse/MOB-167
-		$wgOut->addHeadItem('My Wikia Smart Banner', '<meta name="apple-itunes-app" content="app-id=623705389">');
```

Comment:
Yes, please provide MyWikia as the default if no config is set in WikiFactory (only for iPad, as MyWikia supports iPad only)